src/hotspot/share/oops/constantPool.cpp

```diff
@@ -2213,15 +2213,15 @@ int ConstantPool::copy_cpool_bytes(int cpool_size,
 
 #undef DBG
 
-bool ConstantPool::is_maybe_on_continuation_stack() const {
-  // This method uses the similar logic as nmethod::is_maybe_on_continuation_stack()
+bool ConstantPool::is_maybe_on_stack() const {
+  // This method uses the similar logic as nmethod::is_maybe_on_stack()
   if (!Continuations::enabled()) {
     return false;
   }
 
   // If the condition below is true, it means that the nmethod was found to
```

Comment:
I think this comment should say Method and not nmethod.  I think this was a cut and paste error.  I assume that the is_maybe_on_stack is generalizing "stack" meaning on the execution OR continuation stack ?

Reply:
I think it really does reference nmethod::is_maybe_on_stack(), and explains that the logic for ConstantPool::is_maybe_on_stack works "similar" to how it works for nmethods.